src/migration/1697642145710-TagSearchIndex.ts

```diff
@@ -0,0 +1,14 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class TagSearchIndex1697642145710 implements MigrationInterface {
```

Comment:
Isn't the point of having migrations to do one thing? For example adding this index breaks something. We want to revert it, we can do it easily. If materialized view was bundled with it we would also revert that part which would break rest of the codebase.

Reply:
Ideally, you should have one migration for PR because PR is what gets deployed eventually. A migration runs in a transaction so you can't have a situation where something succeeded and something failed, one split to multiple migration you can have this situation and then you need to check what migration caused the issue. In case the migration fails in production, the deployment will stop and there will be no impact on production anyway